<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>593</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: #e6f2ff;
background-color: #2c2c47;
</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>10</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3"/>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string>color: #e6f2ff;
background-color: #007399;
border-color: #034b33;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QLabel" name="label_login">
         <property name="styleSheet">
          <string>color: #e6f2ff;
background-color: #007399;
border-color: #034b33;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox">
           <property name="styleSheet">
            <string notr="true">color: #e6f2ff;
background-color: #007399;
border-color: #034b33;</string>
           </property>
           <property name="title">
            <string>Sign in</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_username">
                  <property name="styleSheet">
                   <string notr="true">color: #e6f2ff;
background-color: #007399;
border-color: #034b33;</string>
                  </property>
                  <property name="text">
                   <string>Username</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_username">
                  <property name="styleSheet">
                   <string notr="true">color: #e6f2ff;
background-color: #007399;
border-color: #034b33;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_password">
                  <property name="styleSheet">
                   <string notr="true">color: #e6f2ff;
background-color: #007399;
border-color: #034b33;</string>
                  </property>
                  <property name="text">
                   <string>Password</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_password">
                  <property name="styleSheet">
                   <string>color: #e6f2ff;
background-color: #007399;
border-color: #034b33;</string>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_login">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #e6f2ff;
background-color: #007399;
border-color: #034b33;</string>
              </property>
              <property name="text">
               <string>Login</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_reset">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #e6f2ff;
background-color: #007399;
border-color: #034b33;</string>
              </property>
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQuit">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionQuit"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/images/quit.png</normaloff>:/img/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/images/help.png</normaloff>:/img/images/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
